sites/site-utilities/statics/server/server.js

```diff
@@ -48,11 +48,11 @@ app.use(fallback("index.html", { root: publicDir }));
 // Manage search engine crawlers if staging add robots.txt, otherwise delete
 if (process.env.WEBSITE_HOSTNAME.indexOf("-stage") > -1) {
     fs.writeFile("robots.txt", "User-agent: *\r\nDisallow: /", function (err) {
-        if (err) throw err;
+        if (err) console.log(err);
```

Comment:
Do we know why this is erroring? Would it be better to fix the underlying issue than to change a throw to a log?

Reply:
Yes, because the file doesn't exist on production. There are two use cases. Either Staging is promoted to Production and the file gets moved then deleted because it should not exist or the file doesn't exist because we're moving from the master branch directly into production without first going to staging.  Both are by design.